project/assets/new-evaluation.less

```diff
@@ -0,0 +1,91 @@
+.new-eval {
+    margin: auto;
+    margin-top: 5%;
+    width: 70%;
+}
+
+textarea {
+    width: 100%;
+    border: 0.5px solid black;
+}
+
+input[type=date],
+input[type=text] {
+    width: 30%;
+    display: inline-block;
+    outline: 0;
+    border-width: 0 0 2px;
+    border-color: rgb(166, 166, 166);
+}
+
+input[type=radio] {
+    border-color: rgb(166, 166, 166);
+}
+
+table {
+    border-collapse: collapse;
+    width: 100%;
+    margin-top: 0%;
+}
+
+.new-eval-button {
+    box-shadow: 10px 35px -9px rgba(204, 194, 204, 1);
+    line-height: 52px;
+    width: 13%;
+    margin-left: 90%;
+    margin-top: 4%;
+}
+
+.new-eval-button-skin {
+    background-color: rgb(51, 153, 255);
+    color: rgb(255, 255, 255);
+    border: none;
+    font-family: roboto;
+    font-size: 20px;
+}
+
+option,
+select {
+    font-size: x-large;
+    color: rgb(77, 77, 77);
+    width: 7%;
+    text-align: center;
+    margin: 1% 5% 0 4%;
+    display: inline-block;
+}
+
+label {
+    font-size: large;
+    display: inline-block;
+    color: rgb(77, 77, 77);
+    margin-left: -4%;
```

Comment:
in my opinion, these are not styles generic enought to be used in a 'label' DOM tag selectors( not all labels in the app probably will have these characteristics). I suggest to use a class.